src/Common/tests/System/Collections/ICollection.Generic.Tests.cs

```diff
@@ -69,45 +69,42 @@ protected override IEnumerable<T> GenericIEnumerableFactory(int count)
         /// <summary>
         /// Returns a set of ModifyEnumerable delegates that modify the enumerable passed to them.
         /// </summary>
-        protected override IEnumerable<ModifyEnumerable> ModifyEnumerables
+        protected override IEnumerable<ModifyEnumerable> GetModifyEnumerables(ModifyOperation operations)
         {
-            get
+            if (!AddRemoveClear_ThrowsNotSupported && (operations & ModifyOperation.Add) == ModifyOperation.Add)
             {
-                if (!AddRemoveClear_ThrowsNotSupported)
+                yield return (IEnumerable<T> enumerable) =>
                 {
-                    yield return (IEnumerable<T> enumerable) =>
-                    {
-                        var casted = (ICollection<T>)enumerable;
-                        casted.Add(CreateT(2344));
-                        return true;
-                    };
-                }
-                if (!AddRemoveClear_ThrowsNotSupported)
+                    var casted = (ICollection<T>)enumerable;
+                    casted.Add(CreateT(2344));
+                    return true;
+                };
+            }
+            if (!AddRemoveClear_ThrowsNotSupported && (operations & ModifyOperation.Add) == ModifyOperation.Remove)
```

Comment:
Surely this is a typo?  See also line 96

Reply:
Yes, fixed, thanks.